### PR TITLE
chore: reformat scripts and config

### PR DESCRIPTION
**Pull request by SkyeYoung**
Changes:

- reformat scripts and config
- add some missing eslint and git rules

Screenshots of the change:

none

### CODE REVIEW

File: scripts/common.js [hunk 1]
```diff
@@ -1,60 +1,58 @@
...
-    hasChangelog: true
-  }
+    name: 'apisix-python-plugin-runner',
+    branch: 'master',
+    hasChangelog: true,
+  },
 ];
 
-const languages = ["en", "zh", "es"];
+const languages = ['en', 'zh', 'es'];
```
Comment (yzeng25) [inline scripts/common.js:44]: I think we don't need es(Spanish) for now. There was a contributor doing Spanish and he passed away about a year ago, then the process of translation is stopped, see https://github.com/apache/apisix/issues/3587#issuecomment-818397330.

I  would suggest to transfer this part as an issue and see if anyone could pick it up.

```suggestion
const languages = ['en', 'zh'];
```

Comment (SkyeYoung) [reply scripts/common.js:44]: > I would suggest to transfer this part as an issue and see if anyone could pick it up.

🤣 Let's create an issue.

Comment (SkyeYoung) [reply scripts/common.js:44]: > 🤣 Let's create an issue.

I've created https://github.com/apache/apisix-website/issues/1003

Comment (yzeng25) [reply scripts/common.js:44]: Cool. Do you want to keep it as `const languages = ['en', 'zh', 'es'];` for now? I think it is better this way.

Comment (SkyeYoung) [reply scripts/common.js:44]: @yzeng25 I did not change it, i just create an issue. :D

Comment (yzeng25) [reply scripts/common.js:44]: Ok.

**Review by yzeng25 — APPROVED** [review]
LGTM

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**